torch/csrc/autograd/saved_variable.cpp

```diff
@@ -78,11 +78,12 @@ Variable SavedVariable::unpack(std::shared_ptr<Function> saved_for) const {
   // NB: saved views are unpacked as normal Variables (not views) even though
   // they still share the same storage. This works only because we never call
   // in-place functions on unpacked variables.
-  Variable var;
+  Variable var = as_variable_ref(data_).variable_data();
+  var.unsafeGetTensorImpl()->set_allow_tensor_metadata_change(true);
```

Comment:
We set `var`'s `allow_tensor_metadata_change` to true here, because we should not restrict any resizing / restriding ops on the reconstructed Variable.